.travis.yml

```diff
@@ -1,24 +1,32 @@
-language: general
-sudo: required
-dist: trusty
-before_install:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-install:
-  - sudo apt-get update -y
-  - sudo apt-get install python-dev libffi-dev libssl-dev -y
-  - sudo apt-get install python-virtualenv lua5.1 -y
-  - sudo add-apt-repository -y ppa:bartbes/love-stable
-  - sudo apt-get update -y
-  - sudo apt-get install -y love
-  - mkdir -p $TRAVIS_BUILD_DIR/share/love/
-env:
-  global:
-  - XDG_DATA_HOME="$TRAVIS_BUILD_DIR/share"
-script: make deploy
+language: generic
+os:
+  - linux
+dist: bionic
 addons:
+  apt:
+    packages:
+      - xvfb
+      - python-dev
+      - libffi-dev
+      - libssl-dev
+      - python-virtualenv
+      - lua5.1
+      - libpulse0
+      - pulseaudio
+      - sndiod
+      - virtualenv
+      - libsdl2-2.0-0
+      - alsa
+      - alsa-tools
   artifacts:
     paths:
       - ./build/hawkthorne-osx.zip
-      - ./build/hawkthorne-win-x86.zip
+      - ./build/hawkthorne-win-x86_64.zip
       - ./build/hawkthorne.love
+env:
+  global:
+  - XDG_DATA_HOME="$TRAVIS_BUILD_DIR/share"
+script:
+  - "export DISPLAY=:99.0"
+  - "systemctl start xvfb"
+  - "make deploy"
```

Comment:
Probably want `xvfb-run`

Reply:
have make this : xvfb-run :99 -screen 0 1024x768x24

Reply:
Not work... damm

Reply:
Only need xvfb-run